src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -185,9 +186,7 @@ public async Task<ActionResult> Setup(SetupApiViewModel model)
                 return BadRequest(S["The database provider is not defined."]);
             }
 
-            var selectedProvider = _databaseProviders.FirstOrDefault(provider =>
-                String.Equals(provider.Value, databaseProvider, StringComparison.OrdinalIgnoreCase));
-
+            var selectedProvider = _databaseProviders.FirstOrDefault(provider => provider.Value == databaseProvider);
```

Comment:
Why the casing in the comparison is removed

Reply:
Because we check a technical value